### PR TITLE
API-1835: Add input-resources

### PR DESCRIPTION
**Pull request by bertinatto**

### CODE REVIEW

File: pkg/cmd/mom/input_resources_command.go [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+				libraryinputresources.ExactLowLevelOperator("authentications"),
+				// Config
+				libraryinputresources.ExactConfigResource("apiservers"),
+				libraryinputresources.ExactConfigResource("infrastructures"),
+				libraryinputresources.ExactConfigResource("oauths"),
+				libraryinputresources.ExactConfigResource("ingresses"),
+				libraryinputresources.ExactConfigResource("consoles"),
+				libraryinputresources.ExactConfigResource("proxies"),
+				libraryinputresources.ExactResource("config.openshift.io", "v1", "clusterversions", "", "version"),
+				// OpenShiftx
```
Comment (p0lyn0mial) [inline pkg/cmd/mom/input_resources_command.go:29]: nit: typo

File: pkg/cmd/mom/input_resources_command.go [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+	"github.com/openshift/multi-operator-manager/pkg/library/libraryinputresources"
+	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericiooptions"
+)
+
+func NewInputResourcesCommand(streams genericiooptions.IOStreams) *cobra.Command {
+	return libraryinputresources.NewInputResourcesCommand(RunInputResources, streams)
+}
+
+func RunInputResources(ctx context.Context) (*libraryinputresources.InputResources, error) {
```
Comment (p0lyn0mial) [inline pkg/cmd/mom/input_resources_command.go:15]: can be private

File: pkg/cmd/mom/input_resources_command.go [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+				libraryinputresources.ExactConfigMap("openshift-authentication", "v4-0-config-system-metadata"),
+				libraryinputresources.ExactConfigMap("openshift-authentication", "v4-0-config-system-service-ca"),
+				libraryinputresources.ExactSecret("openshift-authentication", "v4-0-config-system-serving-cert"),
+				libraryinputresources.ExactSecret("openshift-authentication", "v4-0-config-system-session"),
+				libraryinputresources.ExactSecret("openshift-authentication", "v4-0-config-system-router-certs"),
+				libraryinputresources.ExactSecret("openshift-authentication", "v4-0-config-system-custom-router-certs"),
+				libraryinputresources.ExactSecret("openshift-authentication", "v4-0-config-system-ocp-branding-template"),
+			},
+		},
+		OperandResources: libraryinputresources.OperandResourceList{
```
Comment (p0lyn0mial) [inline pkg/cmd/mom/input_resources_command.go:52]: @deads2k what is the purpose of this field?

File: pkg/cmd/mom/input_resources_command.go [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericiooptions"
+)
+
+func NewInputResourcesCommand(streams genericiooptions.IOStreams) *cobra.Command {
+	return libraryinputresources.NewInputResourcesCommand(RunInputResources, streams)
+}
+
+func RunInputResources(ctx context.Context) (*libraryinputresources.InputResources, error) {
+	return &libraryinputresources.InputResources{
```
Comment (p0lyn0mial) [inline pkg/cmd/mom/input_resources_command.go:16]: If we get this list wrong then the operator will fail, right ?

File: pkg/cmd/mom/input_resources_command.go (hunk 2 above)
Comment (bertinatto) [reply pkg/cmd/mom/input_resources_command.go:15]: Done

File: pkg/cmd/mom/input_resources_command.go (hunk 3 above)
Comment (bertinatto) [reply pkg/cmd/mom/input_resources_command.go:52]: As we discussed in our meeting, removed

File: pkg/cmd/mom/input_resources_command.go (hunk 1 above)
Comment (bertinatto) [reply pkg/cmd/mom/input_resources_command.go:29]: Fixed

File: pkg/cmd/mom/input_resources_command.go (hunk 4 above)
Comment (deads2k) [reply pkg/cmd/mom/input_resources_command.go:16]: > If we get this list wrong then the operator will fail, right ?

discussed on the call. to be added.

**Pull request merged**